### PR TITLE
updating Bt's with the selectors

### PR DESCRIPTION
**Pull request by padhupradheep**

### CODE REVIEW

**Review by SteveMacenski — COMMENTED** [review]
Can you make this PR include the migration guide (with a screen shot or gif of the tool) for the new selector feature + all the BTs were updated to be able to use it?

Easier to do this all in batch

**Comment by padhupradheep** [comment]
Tbh, entirely forgot about the migration guide.. got bit rusty :laughing: 

Did the necessary updates..

File: migration/Iron.rst [hunk 1]
```diff
@@ -229,4 +229,23 @@ New Graceful Motion Controller
...
-See :ref:`configuring_graceful_motion_controller` for more information.
\ No newline at end of file
+See :ref:`configuring_graceful_motion_controller` for more information.
+
+New RViz Plugin for selecting Planners, Controllers, Goal Checkers, and Smoothers
+*********************************************************************************
+
+`In PR #4091 <https://github.com/ros-planning/navigation2/pull/4091>`_ a new RViz plugin was added to select the planner, controller, goal checker, and smoother on the fly.
+
+The primarly goal of this plugin is to facilitate the developers and integrators to easily test their configuration before deploying the robot in the intended application.
```
Comment (SteveMacenski) [inline migration/Iron.rst:239]:
```suggestion
The primary goal of this plugin is to facilitate the developers and easy integration testing of their configuration before deploying the robot in the intended application.
```

**Comment by SteveMacenski** [comment]
Pending discussion in ticket

**Review by SteveMacenski — APPROVED** [review]

File: requirements.txt [hunk 2]
```diff
@@ -1,9 +1,9 @@
-breathe==4.28.0
-docutils==0.14
+breathe
```
Comment (SteveMacenski) [inline requirements.txt:1]: Revert, these should stay as they are for this PR

**Review by SteveMacenski — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 2 above)
Comment (padhupradheep) [reply requirements.txt:1]: sorry.. damn!

**Review by SteveMacenski — APPROVED** [review]

**Pull request merged**